packages/kbn-plugin-generator/sao_template/sao.js

```diff
@@ -61,13 +61,19 @@ module.exports = function({ name, targetPath, isKibanaPlugin }) {
         when: answers => answers.generateApp,
         default: true,
       },
+      generateEslint: {
+        type: 'confirm',
+        message: 'Should generate custom eslint file be used?',
```

Comment:
The phrasing is a little confusing here. Maybe something closer to:

```suggestion
        message: 'Would you like to use a custom eslint file?',
```